### PR TITLE
General Fixes on 

### PR DESCRIPTION
**Pull request by Enderchief**
Each change is divided into its own commit and these only apply to the `website` package. I separated it this way so a change can easily be removed.

- lint all with `npx eslint --ext ts src --fix`
- alias imports. (no more `../../../foo/bar.js`)
  ```yaml
  $layouts/*: "src/layouts/*"
  $components/*: "src/components/*"
  $server/*: "src/server/*"
  ```
- `/api/v1/*` was responding with `text/plain` when but giving json objects, this adds a utility function which takes the same arguments as `new Response(...)` but auto adds the header `"content-type": "application/json"`
- I forgot the license for the last commit

### CODE REVIEW

File: packages/website/src/server/database/account.ts [hunk 1]
```diff
@@ -32,27 +32,27 @@ import database from './database.js'
...
-  platform: string
-  id: string
+;export type PronounsOfUser = {
+  pronouns: string;
+  platform: string;
+  id: string;
 }
-
-export async function createAccount (from: ExternalAccount) {
+;export async function createAccount (from: ExternalAccount) {
```
Comment (cyyynthia) [inline packages/website/src/server/database/account.ts]: These semi colons shouldn't be here

File: packages/website/src/server/database/account.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@
  *
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted provided that the following conditions are met:
- *
+
```
Comment (cyyynthia) [inline packages/website/src/server/database/account.ts]: The star shouldn't have been removed here

File: packages/website/src/server/database/account.ts [hunk 3]
```diff
@@ -98,12 +102,22 @@ export async function updatePronouns (userId: ObjectId, pronouns: string) {
   await collection.updateOne({ _id: userId }, { $set: { pronouns: pronouns } })
 }
 
-export async function addLinkedAccount (userId: ObjectId, account: ExternalAccount) {
+export async function addLinkedAccount (
+  userId: ObjectId,
+  account: ExternalAccount
+) {
```
Comment (cyyynthia) [inline packages/website/src/server/database/account.ts]: I hate multiline things like that

File: packages/website/src/server/utils.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+export function responseJSON (
```
Comment (cyyynthia) [inline packages/website/src/server/utils.ts]: I wouldn't mess with that yet. While I wrote it using Astro's base endpoint feature, according to my benchmarks it performs 2x worse than Fastify and I'm considering throwing in `@matthewp/astro-fastify`, which will definitely change how things are done.

lookup and lookup-bulk receive a crap ton of load and I'd rather avoid the performance pitfalls Astro has due to the use of objects like `Request`, `Response`, etc...

File: packages/website/tsconfig.json [hunk 5]
```diff
@@ -1,6 +1,12 @@
 {
   "extends": "astro/tsconfigs/strictest",
   "compilerOptions": {
-    "types": [ "astro/client" ]
+    "types": ["astro/client"],
+    "baseUrl": ".",
+    "paths": {
+      "$layouts/*": ["src/layouts/*"],
```
Comment (cyyynthia) [inline packages/website/tsconfig.json]: `$` seems unusual. Any reason not use `@` which I've seen much more often?

**Review by cyyynthia — CHANGES_REQUESTED** [review]

File: packages/website/tsconfig.json (hunk 5 above)
Comment (Enderchief) [reply packages/website/tsconfig.json]: no reason. i've seen `$` more so it's just a stylistic thing. I can change it. Also one of the justifications I know is using `@` can be confused/conflict with npm packages

**Comment by Enderchief** [comment]
I'll revert all commits (sometime today) except the aliases

File: packages/website/tsconfig.json (hunk 5 above)
Comment (cyyynthia) [reply packages/website/tsconfig.json]: Fair enough. [Astro docs](https://docs.astro.build/en/guides/aliases/) use `@` so I'd stick to that instead

**Comment by Enderchief** [comment]
Fixed! (also removed all unwanted commits and amended this one)

Changes
- alias imports. (no more `../../../foo/bar.js`)
  ```yaml
  "@layouts/*": "src/layouts/*"
  "@components/*": "src/components/*"
  "@server/*": "src/server/*"
  ```

**Review by cyyynthia — CHANGES_REQUESTED** [review]
It seems I created conflicts with the lockfile ugh.. In all cases, it'd be nice if you could also alias `assets`.

Please also put spaces inside the array (`[ "..." ]` instead of `["..."]`)

**Review by cyyynthia — APPROVED** [review]

**Pull request merged**